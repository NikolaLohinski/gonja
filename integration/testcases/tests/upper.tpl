{{ 'GONJA' is upper }}
{{ 'GÃ–NJA' is upper }}
{{ 'gonja' is upper }}
{{ 'Gonja' is upper }}
{{ 42 is upper }}
