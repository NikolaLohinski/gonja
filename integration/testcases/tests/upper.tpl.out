True
True
False
False
False
